### DISCUSSION TITLE
Add config flag to exclude "made-with-panda" CSS Var

### DISCUSSION THREAD

**Discussion by rsmidt** [chakra-ui/panda#1156] [Ideas]
I've just stumbled across the "--made-with-panda" CSS Var that is added to the global styles.

IMO, this is unnecessarily polluting the global variables. Searching through the docs, I couldn't find a way to disable this behavior.

**Comment by Gravy59** [comment]
It's only ~25 bytes total though

**Reply by rsmidt** [reply]
Still, it's not needed to provide the necessary function, and it's basically user tracking without opt-out. Currently, I need to manually filter it out.
